pkg/serverconfig/config.go

```diff
@@ -35,10 +35,17 @@ func SetFlagsFromConfig(fs *flag.FlagSet, filename string) (err error) {
 	addAuth(fs, &config.Auth)
 	addCustomization(fs, &config.Customization)
 	addProviders(fs, &config.Providers)
+	addTrustedCAFile(fs, &config.Proxy)
 
 	return nil
 }
 
+func addTrustedCAFile(fs *flag.FlagSet, proxy *Proxy) {
+	if proxy.TrustedCAFile != "" {
+		fs.Set("ca-file", proxy.TrustedCAFile)
```

Comment:
I believe we should be setting `user-auth-oidc-ca-file`. The only time this should be used currently is when talking to the OAuth server.